server/src/concepts/offer/model/OfferValidation.js

```diff
@@ -0,0 +1,33 @@
+import Joi from "joi-oid";
+
+const validateCreateOffer = (offer) => {
+  const schema = Joi.object({
+    title: Joi.string().min(3).max(50).required(),
+
+    description: Joi.string().max(500).required(),
+
+    city: Joi.string().max(58).required(),
+
+    phoneNumber: Joi.number().min(9).max(9).required(),
```

Comment:
it is wrong too, for now only one accepted value is:
phoneNumber: 9
if type of field is number min and max means accepted value >= min(9) // accepted value <= max(9)
max and min will work  properly if phoneNumber would be astring, but we have to change it in db model as well. We were talking about that last week I think that phoneNumber should be a string so we can change it